srcpkgs/nvidia/template

```diff
@@ -278,8 +278,11 @@ nvidia-libs_package() {
 
 nvidia-dkms_package() {
 	short_desc="${_desc} - DKMS kernel module"
+	depends="dkms xbps-triggers>=0.121_1"
 	dkms_modules="nvidia ${version}"
-	depends="dkms"
+	# dkms must be before initramfs-regenerate to build modules before images
+	triggers="dkms initramfs-regenerate"
```

Comment:
I guess we don't want to do this for all dkms modules? Otherwise it could be automated.

Reply:
I'm open to that argument, but I think ZFS and nvidia are somewhat special cases; for example, `acpi_call-dkms` probably never needs to be folded into an initramfs image, so triggering a rebuild of them would be wasteful. Adding an opt-in trigger is a bit of a compromise (the downside, of course, is that the trigger order matters so we can't rely on `dkms_modules` to automatically insert `dkms` without breaking the ordering).